lib/activerecord/all/activerecord.rbi

```diff
@@ -493,30 +493,23 @@ class ActiveRecord::Base
   )
   end
 
+  # https://github.com/rails/rails/blob/c8910a90f/activemodel/lib/active_model/validations.rb#L131-L152
   sig do
     params(
-      names: T.any(T::Array[Symbol], Symbol),
-      presence: T.nilable(T::Boolean),
-      inclusion: T.any(
-        T::Array[T.any(String, Symbol)],
-        { in: T::Array[T.any(Symbol, String, T::Boolean, NilClass)]}
-      ),
-      unless: T.nilable(Symbol),
-      if: T.nilable(Symbol),
-      allow_nil: T.nilable(T::Boolean),
-      on: T.nilable(Symbol)
+      names: T.any(Symbol, String),
+      if: T.any(Symbol, String, T.proc.params(arg0: T.untyped).returns(T::Boolean)),
+      on: T.any(Symbol, String),
+      prepend: T::Boolean,
+      unless: T.any(Symbol, String, T.proc.params(arg0: T.untyped).returns(T::Boolean)),
     ).void
   end
   def self.validate(
     *names,
-    presence: nil,
-    inclusion: nil,
-    on: nil,
-    unless: nil,
     if: nil,
-    allow_nil: nil
-  )
-  end
+    on: nil,
+    prepend: T::Boolean,
```

Comment:
In the `def`, should this be `prepend: nil`?

Reply:
Thank you!